src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -3537,9 +3537,7 @@ public static void writeStringToFile(final File file, final String data, final S
 
     /**
      * Instances should NOT be constructed in standard programming.
-     * @deprecated Will be private in 3.0.
      */
-    @Deprecated
```

Comment:
Should this `@Deprecated` be removed? There was a comment to remove a different `@Deprecated` but this one should be kept.

Reply:
rollback it

Reply:
-1: Don't remove this tag,

Reply:
this is not resolved - you have still removed a comment that was in the code before - can you put it back?

Reply:
The `@deprecated` tag should NOT be removed.

Reply:
Hi All,
     sorry for my error operation, I fixed the doc and add @deprecated tag about FileUtils.